commands/moderation/verbal.js

```diff
@@ -0,0 +1,166 @@
+const Discord = require('discord.js');
+const dateFormat = require('dateformat');
+const {verifyReasonLength} = require('../../helpers/stringHelpers');
+
+module.exports = {
+  name: 'verbal',
+  description: 'sends a user a verbal warning and logs note in db',
+  guildOnly: true,
+  staffOnly: true,
+  minRole: 'Moderator',
+  execute(msg, args, con) {
+    // Make sure only SU, Mods and Admin can run the command
+    const offendingUser =
+      msg.mentions.members.first() || msg.guild.members.cache.get(args[0]);
+
+    if (
+      hasUserTarget(msg, offendingUser) &&
+      notSelf(msg, offendingUser) &&
+      notHighRoller(msg, offendingUser)
+    ) {
+      // Parse the reason for the warning
+      // if no reason provided, return so the bot doesn't go boom
+      const verbalReason = args.slice(1).join(' ');
+      if (!verbalReason) return msg.reply('You need to provide a reason');
+
+      if (!verifyReasonLength(msg.content, msg)) return;
+
+      // Create an embed, craft it, and DM the user
+      try {
+        dmTheUser(msg, offendingUser, verbalReason);
+      } catch (e) {
+        return msg.reply('Unable to send a DM to the specified user.');
+      }
+
+      // Register call in the Audit-log channel
+      auditLog(msg, offendingUser, verbalReason);
+
+      // Give SU, Mod, Admin feedback on their call
+      msg.channel.send(`${msg.author} just verballed ${offendingUser}`);
+
+      // Add the infraction to the database
+      recordInDB(msg, con, offendingUser, verbalReason);
+    }
+  },
+};
+
+function auditLog(message, targetUser, reason) {
+  // Outputs a message to the audit-logs channel.
+  const channel = message.guild.channels.cache.find(
+    (channel) => channel.name === 'audit-logs'
+  );
+
+  const verbalEmbed = new Discord.MessageEmbed()
+    .setColor('#f1d302')
+    .setTitle(
+      `${targetUser.user.username}#${targetUser.user.discriminator} was verballed by ${message.author.tag}:`
+    )
+    .setDescription(reason)
+    .setThumbnail(
+      `https://cdn.discordapp.com/avatars/${targetUser.user.id}/${targetUser.user.avatar}.png`
+    )
+    .setTimestamp()
+    .setFooter(`${message.guild.name}`);
+
+  channel.send(verbalEmbed);
+}
+
+function dmTheUser(msg, targetUser, reason) {
+  // Create an embed, craft it, and DM the user
+  const Embed = new Discord.MessageEmbed()
+    .setColor('#f1d302')
+    .setTitle(`Verbal to ${targetUser.user.username}`)
+    .addField(
+      '',
```

Comment:
Got the following error from having an empty string here:
`RangeError [EMBED_FIELD_NAME]: MessageEmbed field names may not be empty.`